README.md

```diff
@@ -1,14 +1,42 @@
-# Code Kata
+# Fenders Perth Community Kata
 
-This repo will contain various code kata challenges.
+## What is a kata?
 
-# Folder Structure
+Code Kata is an attempt to bring this element of practice to software development. A kata is an exercise in karate where you repeat a form many, many times, making little improvements in each. The intent behind code kata is similar. Each is a short exercise (perhaps 30 minutes to an hour long). Some involve programming, and can be coded in many different ways. Some are open ended, and involve thinking about the issues behind programming. These are unlikely to have a single correct answer. I add a new kata every week or so. Invest some time in your craft and try them.
 
-For now, I propose people maintain their own folders containing their username and kata name. Eventually it might be handy to further namespace with the current week of the year but for now keep it simple.
+**Remember that the point of the kata is not arriving at a correct answer. The point is the stuff you learn along the way. The goal is the practice, not the solution.**
 
-Example kata path:
+Find out more about [code kata here](http://codekata.com/).
+
+
+## Previous kata exercises
+
+1. [Angry Professor](https://www.hackerrank.com/challenges/angry-professor)
+2. [Cut The Sticks](https://www.hackerrank.com/challenges/cut-the-sticks)
+3. [Cavity Map](https://www.hackerrank.com/challenges/cavity-map)
+4. [Chocolate Feast](https://www.hackerrank.com/challenges/chocolate-feast)
+
+## How to get started
+
+Make sure you're a part of the Fenders Perth organisation on GitHub, this should probably mean you're already a part of our Slack group, but just in case, [here's the link to do that](http://fendersslack.herokuapp.com/).
```

Comment:
No, but it's a good way to get people involved if they're not already ;)

On Apr 7 2016, at 11:38 am, Kevin Tjiam &lt;notifications@github.com&gt;
wrote:  

> In [README.md](https://github.com/fendersperth/kata/pull/22#discussion_r5881
5243):

>  
>  
>     >  
>     > -Example kata path:

>     > +Find out more about [code kata here](http://codekata.com/).

>     > +

>     > +

>     > +## Previous kata exercises

>     > +

>     > +1. [Angry Professor](https://www.hackerrank.com/challenges/angry-
professor)

>     > +2. [Cut The Sticks](https://www.hackerrank.com/challenges/cut-the-
sticks)

>     > +3. [Cavity Map](https://www.hackerrank.com/challenges/cavity-map)

>     > +4. [Chocolate Feast](https://www.hackerrank.com/challenges/chocolate-
feast)

>     > +

>     > +## How to get started

>     > +

>     > +Make sure you're a part of the Fenders Perth organisation on GitHub,
this should probably mean you're already a part of our Slack group, but just
in case, [here's the link to do that](http://fendersslack.herokuapp.com/).

>  
>

> This isn't actually a requirement to get started?

>

> —  
You are receiving this because you authored the thread.  
Reply to this email directly or [view it on GitHub](https://github.com/fenders
perth/kata/pull/22/files/03a16794156d656c720dececcd5dbe5044121a2c#r58815243)![
](https://github.com/notifications/beacon/AABQCvDemGJx67EyzVK9ZWObw0jb2VjDks5p
1HwVgaJpZM4IBs1k.gif)